pymc/tests/distributions/test_continuous.py

```diff
@@ -618,10 +618,13 @@ def test_pareto(self):
         reason="Fails on float32 due to numerical issues",
     )
     def test_weibull_logp(self):
+        # SciPy has new (?) precision issues at {alpha=20, beta=2, x=100}
+        # We circumvent it by skipping alpha=20:
+        rplusbig = Domain([0, 0.5, 0.9, 0.99, 1, 1.5, 2, np.inf])
         check_logp(
             pm.Weibull,
             Rplus,
-            {"alpha": Rplusbig, "beta": Rplusbig},
+            {"alpha": rplusbig, "beta": Rplusbig},
```

Comment:
I tested against the latest Scipy release using `n_samples=-1` and I couldn't reproduce the failure with the old `Rplusbig`

Reply:
But it failed in the CI of the previous commit: https://github.com/pymc-devs/pymc/actions/runs/4000543274/jobs/6866418421

...and that was already the 2nd try of that job. According to the log this was also SciPy 1.10.0

Reply:
Well that is definitely an extreme value regardless of changes in Scipy (e^33), so I think it's fine to use a less extreme grid. Not clear if it will fix it though since we couldn't reproduce it locally.